File: C Modules/Assignments/Assignment no 6/Question-1/Type_1/Assignment no 1/Question_4.c
#include<stdio.h>
void vowel();
int main()
{
    vowel();
    return 0;
}
void vowel(){
        char ch;
    printf("Enter the character:\n");
    scanf("%c",&ch);
    if(ch=='a'||ch=='e'||ch=='i'||ch=='o'||ch=='u'||ch=='A'||ch=='E'||ch=='I'||ch=='O'){
        printf("'%c' is a vowel",ch);
    }
    else{
        printf("'%c' is consonant",ch);
    }
}

File: Java Module/Assignments in c to java/Assignment-01/Question_4.java
public class Question_4 {

    public static void main(String[] args) {
        
    char ch ='o';
    // printf("Enter the character:\n");
    // scanf("%c",&ch);
    if(ch=='a'||ch=='e'||ch=='i'||ch=='o'||ch=='u'||ch=='A'||ch=='E'||ch=='I'||ch=='O'){
        System.out.printf("'%c' is a vowel",ch);
    }
    else{
        System.out.printf("'%c' is consonant",ch);
    }
    }
}